egs2/dns_icassp22/enh1/README.md

```diff
@@ -0,0 +1,19 @@
+# Deep Noise Suppression (DNS) Challenge 4 @ ICASSP 2022
+
+## Environments
+- date: `Wed Sep 14 09:01:46 UTC 2022`
+- python version: `3.9.12 (main, Jun  1 2022, 11:38:51)  [GCC 7.5.0]`
+- espnet version: `espnet 202207`
+- pytorch version: `pytorch 1.12.1+cu102`
+- Git hash: `13db69d3befc3c82a5ff5a11e28bf79d5030603f`
+  - Commit date: `Mon Aug 29 13:44:35 2022 +0000`
+
```

Comment:
how about uploading the model and add it here?

Reply:
Sure. I'm waiting for the model to train on 48kHz, full dataset (which might take several days more)